src/main/java/UI/RegistrationPage.java

```diff
@@ -138,52 +149,103 @@ public RegistrationPage() {
         frame.setSize(1000,1000);
         frame.setLayout(null);
         frame.setVisible(true);
+        frame.setLocationRelativeTo(null);
 
         setTitle("REGISTRATION FORM");
     }
 
     @Override
     public void actionPerformed(ActionEvent ae) {
+
         String username_value = username_textField.getText();
         String name_value = name_textField.getText();
         String age_value = age_textField.getText();
-        String pronouns_value = pronouns_textField.getText();
+        String pronouns_value = "";
+        if (pronouns_comboBox.getSelectedIndex() == 0) {
+            pronouns_value = "He/Him";
+        } else if (pronouns_comboBox.getSelectedIndex() == 1) {
+            pronouns_value = "She/Her";
+        } else if (pronouns_comboBox.getSelectedIndex() == 2) {
+            pronouns_value = "They/Them";
+        }
         String country_value = country_textField.getText();
         String province_value = province_textField.getText();
         String city_value = city_textField.getText();
-        String gender_value = gender_textField.getText();
-        String sexuality_value = sexuality_textField.getText();
-        String interest_value = interest_textField.getText();
+        String gender_value = "";
+        if (gender_comboBox.getSelectedIndex() == 0) {
+            gender_value = "M";
+        } else if (gender_comboBox.getSelectedIndex() == 1) {
+            gender_value = "F";
+        } else if (gender_comboBox.getSelectedIndex() == 2) {
+            gender_value = "N";
+        }
+        String sexuality_value = "";
+        if (sexuality_comboBox.getSelectedIndex() == 0) {
+            sexuality_value = "G";
```

Comment:
This might be smelly code. I have a suggestion:
Can you declare a constant (maybe in another file) which is a mapping from the index value to the right sexuality value? It seems to me that these are all things which can be "shrunk down" with the use of a mapping, so that it's more clean and more easy to read. Alternatively, you don't need a mapping - instead, you can use an ArrayList and use indices to refer to particular interests or the like.